docker/cygnus-ngsi/agent.conf

```diff
@@ -180,6 +182,19 @@ cygnus-ngsi.sinks.orion-sink.keystone_ssl = false
 cygnus-ngsi.sinks.orion-sink.orion_fiware =
 cygnus-ngsi.sinks.orion-sink.orion_fiware_path =
 
+
+# SlicingDiceSink configuration
```

Comment:
New configuration parameter (i.e. cygnus-ngsi.sinks.slicingdice*)  should be documented in the proper documentation file (more on this in a general comment I'll provide soon).